index.html

```diff
@@ -4,15 +4,189 @@
+          <h1 class="headline_header">
+            #1 Javascript <br />
+            Conference On the Planet
+          </h1>
+          <p class="headline_description">
+            Join us, in the first Nigerian edition, of the most prestigious
+            JavaScript conference. <br />
+            2days of international speakers, community, learning and
+            connections.
+          </p>
+          <h4 class="headline_time">2023.02.25(Wed) ~ 27(Fri)</h4>
+          <h4 class="headline_venue">@ Eko Hotels, Lagos, Nigeria</h4>
+        </article>
+      </section>
+
+      <section id="main_program">
+        <article class="program">
+          <h1 class="program_header">Main Program</h1>
+          <span class="program_header_underline"></span>
+          <article class="program_list">
+            <div class="program_item">
+              <span class="material-icons icons_material" id="icons_material">
+                comment_bank
+              </span>
+              <h3 class="program_title">Lecture</h3>
+              <p class="program_description">
+                Listen to the speakers from various countries about the messages
+                of sharing and opening
+              </p>
+            </div>
+            <div class="program_item">
+              <span class="material-icons" id="icons_material">
+                connected_tv
+              </span>
+              <h3 class="program_title">JS Exhibition</h3>
+              <p class="program_description">
+                Appreciate various creations applying JS license of artists,
+                organized from Art Center Lagos.
+              </p>
+            </div>
+            <div class="program_item">
+              <span class="material-icons icons_material" id="icons_material">
+                forum
+              </span>
+              <h3 class="program_title">Forum</h3>
+              <p class="program_description">
+                Have the time to share your thoughs and opinions with the
+                experts for each topic
+              </p>
+            </div>
+            <div class="program_item">
+              <span class="material-icons icons_material" id="icons_material">
+                business_center
+              </span>
+              <h3 class="program_title">Workshop</h3>
+              <p class="program_description">
+                Try creating your own work using open source license rather than
+                just watching
+              </p>
+            </div>
+            <div class="program_item">
+              <span class="material-symbols-outlined" id="icons_material">
+                light_mode
+              </span>
+              <h3 class="program_title">JS Ignite</h3>
+              <p class="program_description">
+                Get opportunities to network with JS affiliates around the
+                world, also after the summit
+              </p>
+            </div>
+          </article>
```

Comment:
- [ ] As from the design template, kindly make sure your cards have some interactions animations once one of them is hovered.
![Screenshot from 2023-01-03 22-40-00](https://user-images.githubusercontent.com/83042742/210438176-c2446cb9-4f5d-4189-9f93-e66eff446ee1.png)